bgpd/bgp_mpath.c

```diff
@@ -120,13 +121,18 @@ int bgp_info_nexthop_cmp(struct bgp_info *bi1, struct bgp_info *bi2)
 					&bi2->attr->mp_nexthop_global);
 				break;
 			case BGP_ATTR_NHLEN_IPV6_GLOBAL_AND_LL:
```

Comment:
I wasn't sure and I didn't have a test case with something I could point at to show things not working quite right so I left it alone